packages/main-ui/src/pages/deliberations/new/details/discussions/components/member.rs

```diff
@@ -0,0 +1,62 @@
+use bdk::prelude::*;
+use models::{DeliberationDiscussionCreateRequest, OrganizationMemberSummary};
+
+use crate::{
+    components::expandable_card::ExpandableCard,
+    pages::deliberations::new::{
+        components::committee_dropdown::CommitteeDropdown,
+        details::discussions::i18n::DiscussionMemberTranslate,
+    },
+};
+
+#[component]
+pub fn DiscussionMember(
+    lang: Language,
+    discussion: DeliberationDiscussionCreateRequest,
+    set_discussion: EventHandler<DeliberationDiscussionCreateRequest>,
+
+    total_committees: Vec<OrganizationMemberSummary>,
+    selected_committees: Vec<OrganizationMemberSummary>,
+) -> Element {
+    let tr: DiscussionMemberTranslate = translate(&lang);
+
+    let select_ids: Vec<i64> = selected_committees.clone().iter().map(|v| v.id).collect();
+
+    rsx! {
+        ExpandableCard { required: false, header: tr.title, description: tr.description,
+            CommitteeDropdown {
+                id: "discussion-committee",
+                hint: tr.search_committee,
+                selected_committees,
+                committees: total_committees,
+
+                add_committee: {
+                    let mut select_ids = select_ids.clone();
+                    let mut discussion = discussion.clone();
+                    move |member: OrganizationMemberSummary| {
+                        select_ids.push(member.id);
+                        discussion.users = select_ids.clone();
+                        set_discussion.call(discussion.clone());
+                    }
+                },
```

Comment:
이런 패턴 사용을 가급적이면 하지 말아주세요.
지금 clone 을 남발하고 있는데, 이러면 메모리 엄청 쓰게 됩니다.

기본적으로는 아래와 같은 형태가 되어야 합니다.

지금도 사실 뭔가 설계가 조금 잘못된거 같긴해보여요. 너무 큰데이터가 이벤트로 왔다갔다 하는거처럼 보이는데요.

일단은 들어오는 파라미터를 갑자기 여기에서 mut 으로 바꿔서 뭔가 작업을 해준다면, 이 컴포넌트 전체는 계속 들어오는 파라미터에 의존할 뿐만 아니라, 랜더링도 계속 전체 re-rendering 이 되게 됩니다.


```suggestion
                add_committee:      move |member: OrganizationMemberSummary| {
                        select_ids.push(member.id);
                        discussion.users = select_ids.clone();
                        set_discussion.call(discussion.clone());
                },
```